docs/source/tutorials/mantid_basic_course/connecting_data_to_instruments/03_investigating_data.rst

```diff
@@ -132,6 +132,8 @@ of them.
 
 Only translation is possible for a multiple selection.
 
+It is possible to copy/paste selected shapes using Ctrl+C / Ctrl+V
```

Comment:
I would add a step to do this, like "try copying x 3 times" else people skim without trying